index.html

```diff
@@ -311,14 +311,16 @@ <h3 class="bannerbox__title">I can offer a ride</h3>
                         <a class="close-form button--cancel" href="#intro">&times;</a>
 
                         <fieldset class="date-time-pickers">
-                            <legend>When and where can you pick someone up?</legend>
+                            <legend>What can you offer?</legend>
 
                             <div class="form-group date-container">
-                                <label for="driver-date">Date available</label>
+                                <label for="driver-date">Dates available</label>
+                                <p>On what date(s) would you be a available to give rides?" We'll propose matches for all of the dates that you pick.</p>
```

Comment:
It looks like the extra double quotation mark after `give rides?` was mistakenly included in the copy supplied in the issue. It doesn't make sense here, so do you mind removing it please? Sorry, this is not your fault.

CC @sasjkia